clusterloader2/cmd/clusterloader.go

```diff
@@ -62,6 +62,7 @@ func initClusterFlags() {
 	flags.IntEnvVar(&clusterLoaderConfig.ClusterConfig.Nodes, "nodes", "NUM_NODES", 0, "number of nodes")
 	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.Provider, "provider", "PROVIDER", "", "Cluster provider")
 	flags.IntEnvVar(&clusterLoaderConfig.ClusterConfig.EtcdInsecurePort, "etcd-insecure-port", "ETCD_INSECURE_PORT", 2382, "Inscure http port")
+	flags.BoolEnvVar(&clusterLoaderConfig.ClusterConfig.DeleteStaleNamespace, "delete-stale-namespace", "DELETE_STALE_NAMESPACE", false, "Whether to delete all stale namespaces before the test execution.")
```

Comment:
Sorry one more nit:
probably should be "DeleteStaleNamespaces" (and delete-stale-namespaces) [i.e. plural namespaces]

Reply:
never mind, good catch!